src/Microsoft.PowerShell.Commands.Management/commands/management/NewPropertyCommand.cs

```diff
@@ -175,4 +182,124 @@ protected override void ProcessRecord()
         #endregion Command code
 
     }
+
+    /// <summary>
+    /// Provides argument completion for PropertyType parameter.
+    /// </summary>
+    public class PropertyTypeArgumentCompleter : IArgumentCompleter
```

Comment:
Please put all code below under `#if !UNIX`

Reply:
Fixed in 0e9900b01